### DISCUSSION TITLE
I don't know how to change the scoreboard

### DISCUSSION THREAD

**Discussion by iPBGyt** [SirBlobman/CombatLogX#682] [General]
How do I change what the scoreboard looks like when showing the combat timer?

**Comment by iPBGyt** [comment]
also how do i make it so i can open chests in combat?

**Comment by SirBlobman** [comment]
You can change the scoreboard in your selected language file (`en_us.lang.yml` is the default). Your other question was answered in the other discussion you created.

**Reply by iPBGyt** [reply]
found that 2 tysm
